config/prometheus/prometheus.yml

```
@@ -0,0 +1,115 @@
+    static_configs:
+      - targets: ['cadvisor:8080']
+    scrape_interval: 15s
+
+  # Blackbox Exporter for endpoint monitoring
+  - job_name: 'blackbox-http'
+    metrics_path: /probe
+    params:
+      module: [http_2xx]
+    static_configs:
+      - targets:
+        - http://snaproom-laravel:8000/api/health
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)